test/e2e/test/elasticsearch/steps_mutation.go

```diff
@@ -156,12 +159,12 @@ func (b Builder) MutationTestSteps(k *test.K8sClient) test.StepList {
 				Test: func(t *testing.T) {
 					continuousHealthChecks.Stop()
 
-					for _, f := range continuousHealthChecks.Failures {
-						t.Errorf("Elasticsearch cluster health check failure at %s: %s", f.timestamp, f.err.Error())
-					}
+					log.Info("continuousHealthChecks %d failures (tolerated %d)",
```

Comment:
Will this templating work? I thought `logr.Logger` takes keys and values?

Reply:
No way it can work 🤦 .
It was probably the joy of finally understanding what was wrong that made me write nonsense.